web-api/src/v1/v1ApiWrapper.js

```diff
@@ -20,7 +20,7 @@ class v1ApiError extends Error {
       : 500;
     this.message = {
       message: error.message || 'An unexpected error occurred',
-      toString: function () {
+      toString() {
```

Comment:
does this also want to be `toString: () => {`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Working_with_Objects#defining_methods

Ah, ok, nm